res/skins/QMLDemo/ResettableControlKnob.qml

```diff
@@ -0,0 +1,25 @@
+import "." as Skin
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+
+Skin.ControlKnob {
+    id: root
+
+    property string resetGroup: group
+    property string resetKey: key + "_set_default"
```

Comment:
maybe instead of relying on `_set_default` of a ControlPotMeter control, it would be better to expose `ControlProxyt::reset()` to the qmlcontrolproxy?

Reply:
This is just a default value, you can override `resetKey` if you want.

Reply:
Right, but my point is that COs already seem have a concept of resetting themselves (`slot ControlProxy::reset()`, `ControlObject::reset()`) so it would make sense to expose that same slot on the QML proxy as well. That would be better than relying on the fact that there is some CO variant whose only purpose is to reset the control. The `_set_default` variant seems like a workaround that was needed because of the legacy skin and controller mapping system (before there was even JS engine).

Reply:
Good point. I'll look into that in a follow up PR, this one is already pretty large. I don't want to add C++ changes, too.

Reply:
I also spend some time on that already :)